autofit/database/model/common.py

```diff
@@ -0,0 +1,36 @@
+from typing import Union
+
+from ..sqlalchemy_ import sa
+
+from autofit.mapper.prior import abstract
+from autofit.mapper.prior_model import prior_model
+from autofit.mapper.prior_model import collection
+
+from .model import Object
+
+
+class Dict(Object):
+    """
+    A dictionary
+    """
```

Comment:
The docstring is not so useful. Perhaps describe why this is different of a dictionary or why you define a dictionary :-)

Reply:
Haha fair point. This was a copilot contribution